curriculum/challenges/english/25-front-end-development/quiz-javascript-audio-and-video/66edd3b3096349f06cf688bb.md

```diff
@@ -264,59 +264,59 @@ Sets the audio to repeat indefinitely
 
 #### --text--
 
-Which method is used to capture a media stream from a `canvas` element?
+What is the full meaning of MIME?
 
 #### --distractors--
 
-`canvas.getStream()`
+Multiple Internet Message Extensions
 
 ---
 
-`canvas.captureMedia()`
+Mail Internet Multimedia Extensions
 
 ---
 
-`canvas.toDataURL()`
+Multipurpose Internet Mail Extensions
 
 #### --answer--
 
-`canvas.captureStream()`
+Multimedia Internet Mail Extensions
 
 ### --question--
 
 #### --text--
 
-What is the purpose of the `MediaRecorder` API?
+What does the `canPlayType()` method help you do? 
 
 #### --distractors--
 
-To play audio and video files
+Check if audio is currently playing in the browser
 
 ---
 
-To edit existing media files
+Verify if a media file exists in the browser
 
 ---
 
-To create visual representations of audio
+Test if an audio element has controls enabled
 
 #### --answer--
 
-To record audio and video from a media stream
+Determine if a browser is likely to be able to play your specific audio format
```

Comment:
I would suggest making the distractors longer so it is not as obvious what the correct answer is.